content/tools/cli/Cli-tool.md

```diff
@@ -0,0 +1,107 @@
+ * [.NET Core agent](installation-netcoreinstall.html) 
+ * [Node agent](installation-nodeinstall.html)
+ 
+* Source code for Java, .NET and Node applications must be available locally to be examined by the CLI tool: 
+
+ * Java applications must have Maven installed with the dependencies plugin. 
+ * .NET and .NET Core applications must have a *package.lock.json* file present.
+ * Node applications must have a *package-lock.json* file present.
+ 
+* Node.js must be installed. (The Contrast SCA CLI tool is executed as a Node.js package.) For information on how to install Node.js, please refer to their [documentation](https://nodejs.org/en/download/). 
+
+### Credentials
+ 
+* Gather the following Contrast credentials: 
+
+ * API key
+ * Organization ID
+ * Contrast URL
+
+To identify your credentials, see the [Profile Settings](user-account.html#profile) article.  
+
+* Identify the [application ID](user-appsmanage.html) of the application you want to analyze. 
+
+
+## Install 
+
+Install the tool with the following command: 
+
+```
+npm install -g @contrast/contrast-sca 
+```
+
+Or, you can install the tool with Yarn with the following command: 
+
+```
+yarn global add @contrast/contrast-sca 
+```
+
+## Authenticate and Analyze 
+
+Once the CLI tool is installed, you can analyze applications and see the results in the Contrast UI. Contrast recommends that SCA is invoked as part of a CI pipeline so that running the SCA is automated as part of your build process. 
+
+Use the commands shown in the following example to analyze an application: 
+
+```
+contrast-sca \ 
+
+ --api-key someAPIKey \ 
+
+ --authorization someAuthorizationCredentials \ 
+
+ --application-id someApplicationID \ 
+
+ --organization-id someOrganizationID \ 
+
+ --host ContrastUIDomain \ 
+
+ --project-path applicationRootDirectory
+
+```
+
+<!-- 
+### Review 
+
+After you see a SUCCESS message you are now ready to view your dependency tree. (link to library hierarchy) 
+-->
+
+### Help 
+
+The tool includes command line help, including the glossary of commands shown below, which you display with the `--h` option. 
+
+
+| Command                   | Description                                                                               |
+|---------------------------|-------------------------------------------------------------------------------------------|
+| --api-key string          | An agent API key provided by Contrast UI (**required**)                                 |
+| --authorization string    | Agent Authorization credentials provided by Contrast UI (**required**)                |
+| --organization-id string  | The ID of your organization in Contrast UI (**required**)                                 |
+| --application-id string   | The ID of the application cataloged by Contrast UI (**required**)                          |
+| --host string             | The name of the host and (optionally) the port expressed as <host>:<port> (**required**)|
```

Comment:
`--host` isn't required in the current version of the tool. It does default to the following value which we ought to document: `app.contrastsecurity.com`